fairscale/utils/object.py

```diff
@@ -9,6 +9,7 @@
 import torch
 
 
+# FIXME: both of these mechanisms at _least_ double allocate and double copy.
```

Comment:
I think this is related to some all_gather call for objects before pytorch supported it. Newer pytorch version might already support it. If all the supported pytorch version has this, we can remove it. We support pytorch LTS, stable, and also test against a nightly build.